modules/ocf_docker/manifests/deckschrubber.pp

```diff
@@ -0,0 +1,22 @@
+class ocf_docker::deckschrubber {
+  package { 'deckschrubber': }
+
+  file { ['/opt/docker', '/opt/docker/registry']:
```

Comment:
I'd probably put this in `/opt/share/puppet/` since we tend to put other scripts there, like [some backup stuff](https://github.com/ocf/puppet/blob/487324734f4d34d5ae68cb79918ac0455cddc1d7/modules/ocf_backups/manifests/offsite.pp#L6-L16), [some puppet group script](https://github.com/ocf/puppet/blob/487324734f4d34d5ae68cb79918ac0455cddc1d7/modules/ocf/manifests/groups.pp#L3-L8), and [other random scripts](https://github.com/ocf/puppet/blob/487324734f4d34d5ae68cb79918ac0455cddc1d7/modules/ocf_desktop/manifests/xsession.pp#L13-L27), etc. It's also already created for you in `ocf::puppet`, so you can just require it as a resource when placing the script there to make it a little cleaner if you'd like.